bin/git-forgit

```diff
@@ -92,21 +96,30 @@ _forgit_log_format=${FORGIT_LOG_FORMAT:-%C(auto)%h%d %s %C(black)%C(bold)%cr%Cre
 _forgit_log_preview_options="--graph --pretty=format:'$_forgit_log_format' --color=always --abbrev-commit --date=relative"
 _forgit_fullscreen_context=${FORGIT_FULLSCREEN_CONTEXT:-10}
 _forgit_preview_context=${FORGIT_PREVIEW_CONTEXT:-3}
-_forgit_is_file_tracked="(git ls-files {} --error-unmatch) &> /dev/null"
+
+_forgit_is_file_tracked() {
```

Comment:
This one is a bit tricky, because the string contains the `{}` placeholder that fzf replaces. Since this is no longer passed to fzf directly and instead used in the `_forgit_*_preview` functions now, we need to pass the file from fzf to `$_forgit_is_file_tracked` somehow. I am not aware of a way to do this without converting it into a function. Since we replace all usages of it, I think it would be the best to replace it with a function in this PR.

Reply:
Ah, I see, didn't notice that. Shure, then let's keep it as it is.